app_dart/lib/src/service/scheduler.dart

```diff
@@ -359,7 +359,15 @@ class Scheduler {
 
     // Filter builders based on the PR diff
```

Comment:
Add to function docstring to what happens based on files being returned from githubService?